### DISCUSSION TITLE
suricata error

### DISCUSSION THREAD

**Discussion by ktroberson** [Security-Onion-Solutions/securityonion#8326] [Unsupported Versions]
I am working on a distributed deployment. On my management node my suricata is not starting it shows an error. I ran docker logs so-suricata and this is the output:

unable to find interface "bond0": no such device

But since this is a manager, do I need to worry about this since suricata runs on the sensors?

**Comment by weslambert** [comment]
Suricata is not allowed to run on the manager, so this is not really an issue. It sounds like you attempted to run it previously?
